tasks/sync-granule/schemas/config.json

```diff
@@ -2,12 +2,12 @@
   "title": "SyncGranuleConfig",
   "description": "Describes the config used by the sync-granule task",
   "type": "object",
-  "required": ["buckets", "provider", "downloadBucket"],
+  "required": [
+    "buckets",
+    "provider",
+    "downloadBucket"
+  ],
   "properties": {
-    "forceDownload": {
```

Comment:
Why are we dropping this property?

Reply:
It was never being used in the code.

Reply:
Seems like we might want to note in the CHANGELOG, as people might still have it on their workflow definitions even though it has no effect. And once it's removed from the config, it will start causing workflow failures